startup/routes.js

```diff
@@ -21,6 +19,8 @@ module.exports = function (app) {
   });
 
   //place routes here ...
-  app.use("/user", authToken, userMealPlanRouter);
+  app.use("/user", userMealPlanRouter, userRouter);
```

Comment:
we should validate if user has permissions for this router so I would suggest to put authToken back here ;)

Reply:
it's not my code